homeassistant/components/plex/sensor.py

```diff
@@ -89,7 +89,9 @@ def update_plex(_):
                 show = await self.hass.async_add_executor_job(sess.show)
                 if show.year is not None:
                     season_title += f" ({show.year!s})"
-                season_episode = sess.seasonEpisode
+                season_episode = await self.hass.async_add_executor_job(
```

Comment:
It would be better to consolidate all session update calls into one function that we schedule on the executor. Now we're switching context at least two times per update. We like to avoid switching back and forth between async and sync context if possible as that is not cheap.